deployment/terraform/agent.go

```diff
@@ -139,11 +139,28 @@ func (t *Terraform) configureAndRunAgents(extAgent *ssh.ExtAgent) error {
 			buf := bytes.NewBufferString("")
 			tpl.Execute(buf, tplVars)
 
+			otelcolConfig, err := fillConfigTemplate(otelcolConfigTmpl, map[string]string{
+				"IncludeFiles": strings.Join([]string{
+					"/home/ubuntu/mattermost-load-test-ng/ltagent.log",
+					"/home/ubuntu/mattermost-load-test-ng/ltcoordinator.log",
+				}, ", "),
```

Comment:
Do these logs (agent and coordinator) get interleaved or is there an easy way to filter one or the other? In other words, I'd expect a separate service name for coordinator's logs.

Reply:
Good point! You can filter them by log name (the panels use that filtering). Do you think that's enough or would you prefer to have a different service name altogether?

Reply:
What the receiver does is `tail`ing the file specified, so I don't think rolled logs is a problem. I'll see how easy it is to add another service to that, it makes sense logically